zproject/urls.py

```diff
@@ -78,7 +79,7 @@
         zerver.views.zephyr.webathena_kerberos_login,
         name='zerver.views.zephyr.webathena_kerberos_login'),
 
-    url(r'^accounts/password/reset/$', password_reset,
+    url(r'^accounts/password/reset/$', rate_limit()(password_reset),
```

Comment:
Yeah, so this protects us against someone who just wants to spam users by sending them password reset emails through Zulip. I get a lot of password reset emails from Facebook :).